src/Chainweb/Mempool/InMem.hs

```diff
@@ -264,12 +264,12 @@ validateOne cfg v badmap now t h =
     txcfg :: TransactionConfig t
     txcfg = _inmemTxCfg cfg
 
-    -- | KILLSWITCH: This is to be removed in a future version of Chainweb. This
-    -- prevents any transaction from entering the mempool.
+    -- | KILLSWITCH: This can be removed once the date itself has passed. Until
```

Comment:
Let's put the concrete date into the comment for easier use of `grep`.

```suggestion
    -- | KILLSWITCH 2019-12-05T16:00:00Z: This can be removed once the date itself has passed. Until
```

Reply:
That date isn't quite right.